Automattic/wp-calypso
# A4A: Implement WPCOM hosting section v3.

This PR implements the new WPCOM hosting section v3 (feature list, testimonials, and client relationship contents).

<img width="1426" alt="Screenshot 2024-12-09 at 6 24 24 PM" src="https://github.com/user-attachments/assets/12adf163-6811-435d-9096-aca9b7c4d400">
<img width="1432" alt="Screenshot 2024-12-09 at 6 24 35 PM" src="https://github.com/user-attachments/assets/bb0a2a11-a74c-4e9f-965b-61d66abecaec">
<img width="1431" alt="Screenshot 2024-12-09 at 6 24 47 PM" src="https://github.com/user-attachments/assets/30092b20-b0c0-478d-ae1b-81ce4ededb2c">


Depends on https://github.com/Automattic/wp-calypso/pull/97162
Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1547
Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1548
Closes https://github.com/Automattic/automattic-for-agencies-dev/issues/1549

## Proposed Changes

* Implement the 'Feature list', 'Testimonials', and 'Client relationship' sections.

## Why are these changes being made?


* This is part of the In-product hosting page v3 project.

## Testing Instructions

* Use the A4A live link and go to the `/marketplace/hosting/wpcom` page.
* Confirm you see the new sections.

## Pre-merge Checklist
